### PR TITLE
Mock OP rewards data

### PR DESCRIPTION
**Pull request by jkadamczyk**
## What changed (plus any additional context for devs)

* Implemented an async function that returns OP rewards mock data after a second

### CODE REVIEW

File: src/screens/op-rewards/mocks/getMockOpData.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    multiplier: {
+      amount: 2,
+      breakdown: [
+        {
+          amount: 1.0,
+          qualifier: 'Base',
+        },
+        {
+          amount: 0.25,
+          qualifier: 'Has previously swapped in Rainbow',
```
Comment (skylarbarrera) [inline src/screens/op-rewards/mocks/getMockOpData.ts:42]: not a big an issue for this case but wonder how we should be doing i18n for strings like this

File: src/screens/op-rewards/OpRewardsSheet.tsx [hunk 2]
```diff
@@ -1,19 +1,38 @@
...
+import { getMockOpData } from '@/screens/op-rewards/mocks/getMockOpData';
 
 export const OpRewardsSheet: React.FC = () => {
   const { height } = useDimensions();
   const { top } = useSafeAreaInsets();
+  const [rewardsData, setRewardsData] = useState<any>(null);
+  const [loading, setLoading] = useState(true);
+
+  useFocusEffect(
+    useCallback(() => {
```
Comment (skylarbarrera) [inline src/screens/op-rewards/OpRewardsSheet.tsx:16]: not super important since this is all mock but should prob use react query yeah?

**Review by skylarbarrera — APPROVED** [review]
💯

**Review by estrattonbailey — APPROVED** [review]
LOVE that this is all in the op-rewards screen, v clean ✨

**Pull request merged**